omas/omas_cocos.py

```diff
@@ -1282,16 +1411,16 @@
 _cocos_signals['waves.coherent_wave.:.profiles_1d.:.e_field_n_tor.:.parallel.phase']='?'                                   #[ADD?]# 3.125000 # e_field  _tor  parallel  [V.m^-1]
 _cocos_signals['waves.coherent_wave.:.profiles_2d.:.e_field_n_tor.:.parallel.amplitude']='?'                               #[ADD?]# 3.125000 # e_field  _tor  parallel  [V.m^-1]
 _cocos_signals['waves.coherent_wave.:.profiles_2d.:.e_field_n_tor.:.parallel.phase']='?'                                   #[ADD?]# 3.125000 # e_field  _tor  parallel  [V.m^-1]
-_cocos_signals['waves.coherent_wave.:.full_wave.:.b_field.parallel.:.values']='?'                                          #[ADD?]# 3.000000 # b  b_field  parallel  [V.m^-1]
+_cocos_signals['waves.coherent_wave.:.full_wave.:.b_field.parallel.:.values']='?'                                          #[ADD?]# 3.000000 # b  b_field  parallel  [T]
```

Comment:
@orso82 this change came from the `make cocos` script and seems to be the only thing it really got wrong (aside from comments related to variables with `_pol` in the name).  It's only in the comments so I don't think it is consequential, but do you understand what the issue is?